internal/sysfs/poll_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build windows || linux || darwin
```

Comment:
this was missing. we really need to finish #1578 because it is very easy to break it more, with each change to things that can use syscalls.